### ISSUE TITLE
Use New Registry API for Context Product Validation

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS-Incubator/validate#696]
## 💡 Description

- https://github.com/NASA-PDS/validate/issues/675
- ~https://github.com/NASA-PDS/validate/issues/592~

**Comment by jordanpadams** [comment]
📆 November status: Blocked by https://github.com/NASA-PDS/registry-api/issues/352. Fix is ready for review as soon as performance is improved. Delaying this update to later in build.

**Comment by jordanpadams** [comment]
📆 February status: Deferring to later build. This is a nice-to-have feature to cutover to using latest API software, but system still supports workaround. No impact to end user for not implementing this.

**Comment by jordanpadams** [comment]
Closing this ticket. The one subtask not completed will go into a future build.
